src/Database/PG/Query/Class.hs

```diff
@@ -23,26 +23,29 @@ where
 
 -------------------------------------------------------------------------------
 
-import Control.Monad.Except
-import Control.Monad.Identity
-import Data.Aeson qualified as J
+import Control.Monad.Except (throwError)
+import Control.Monad.IO.Class (liftIO)
+import Control.Monad.Identity (Identity (..))
+import Control.Monad.Trans.Except (ExceptT (..))
+import Data.Aeson (FromJSON, ToJSON, Value, eitherDecodeStrict, encode)
```

Comment:
Where possible, I've avoided importing functions that modules have re-exported (sometimes from other packages).

I've also gone through and avoided "unnecessary" qualified imports for types that are self-descriptive and commonly used (e.g. `Text` rather than `T.Text`), as well as used qualified imports to explicitly specify certain variants of these common types (e.g. `Lazy.ByteString` rather than `LB.ByteString`).